api/schemas/views.py

```diff
@@ -45,10 +48,49 @@ class RegistrationSchemaDetail(JSONAPIBaseView, generics.RetrieveAPIView):
     required_write_scopes = [CoreScopes.NULL]
 
     serializer_class = RegistrationSchemaSerializer
-    view_category = 'registration-schemas'
+    view_category = 'schemas'
     view_name = 'registration-schema-detail'
 
     # overrides RetrieveAPIView
     def get_object(self):
         schema_id = self.kwargs['schema_id']
         return get_object_or_error(RegistrationSchema, schema_id, self.request)
+
+class RegistrationSchemaFormBlocks(JSONAPIBaseView, generics.ListAPIView):
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    required_read_scopes = [CoreScopes.SCHEMA_FORM_BLOCKS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = RegistrationSchemaFormBlockSerializer
+    view_category = 'schemas'
```

Comment:
Like other `view_category`'s, this should probably be prefixed with `registration-`